### PR TITLE
Add basic ResidualBlock

### PR DESCRIPTION
**Pull request by juliolopezcarballo**

### CODE REVIEW

File: next_sparseconvnet/networks/building_blocks.py [hunk 1]
```diff
@@ -19,17 +19,31 @@ def forward(self, x):
         #f1(x)+f2(x)
         pass
 
-
-class ResidualBlock(torch.nn.Module):
+class ResidualBlock_basic(torch.nn.Module):
```
Comment (mmkekic) [inline next_sparseconvnet/networks/building_blocks.py:36]: add `dim` to `__init__` parameters

File: next_sparseconvnet/networks/building_blocks.py [hunk 2]
```diff
@@ -19,17 +19,31 @@ def forward(self, x):
...
     def __init__(self, inplanes,  kernel):
         # f
         #BNRelu
         #SubmanifoldConvolution
         #BNRELU
         #SubmanifoldConvolution
-        pass
-
+        torch.nn.Module.__init__(self)
+        self.inplanes=inplanes
```
Comment (mmkekic) [inline next_sparseconvnet/networks/building_blocks.py]: This is not needed

File: next_sparseconvnet/networks/building_blocks.py [hunk 3]
```diff
@@ -19,17 +19,31 @@ def forward(self, x):
...
+        self.kernel=kernel
+        #self.eps=eps
```
Comment (mmkekic) [inline next_sparseconvnet/networks/building_blocks.py]: Remove comments

File: next_sparseconvnet/networks/building_blocks.py [hunk 4]
```diff
@@ -19,17 +19,31 @@ def forward(self, x):
...
+        #self.momentum=momentum
+
+        self.bnrelu1=scn.BatchNormReLU(inplanes,eps,momentum)
```
Comment (mmkekic) [inline next_sparseconvnet/networks/building_blocks.py]: Call it bnr1

File: next_sparseconvnet/networks/building_blocks.py [hunk 5]
```diff
@@ -19,17 +19,31 @@ def forward(self, x):
...
+        self.subconv1=scn.SubmanifoldConvolution(3,inplanes,inplanes,(kernel,kernel,kernel),0)
+        self.bnrelu2=scn.BatchNormReLU(inplanes,eps,momentum)
+        self.subconv2=scn.SubmanifoldConvolution(3,inplanes,inplanes,(kernel,kernel,kernel),0)
+        self.suma=scn.AddTable()
```
Comment (mmkekic) [inline next_sparseconvnet/networks/building_blocks.py]: Call it `add`

File: next_sparseconvnet/networks/building_blocks.py [hunk 6]
```diff
@@ -19,17 +19,31 @@ def forward(self, x):
...
 
     def forward(self, x):
         #x+f(x)
-        pass
+        y=self.bnrelu1(x)
```
Comment (mmkekic) [inline next_sparseconvnet/networks/building_blocks.py]: white spaces around `=`
